mocks/WorkflowRun.go

```diff
@@ -21,9 +21,11 @@
 // Code generated by mockery v1.0.0. DO NOT EDIT.
 package mocks
 
-import context "context"
+import (
```

Comment:
did you re-run mocks?

Reply:
goimports rewrites the mocks I think.  which should be fine, mockery just can't produce it correctly from scratch.

but I'll double check.

Reply:
yeah, it's just goimports.
#1239 may give us a standard way to generate them (though I'd like to remove them for v2), which will also give us an easy way to format them up-front so it's not forgotten